samples/peripheral/radio_test/sysbuild/remote_shell/pin_fwd.dts

```diff
@@ -1,15 +1,16 @@
 /*
- * Copyright (c) 2022 Nordic Semiconductor ASA
+ * Copyright (c) 2024 Nordic Semiconductor ASA
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-/* Forwards nRF21540 EK pins to the network core. Network core requires these pins
- * to control front-end module by GPIO and SPI interfaces.
+/* Forwards nRF21540 EK and radio dfe pins to the network core. Network core requires these pins
+ * to control front-end module by GPIO and SPI interfaces and to perform the DFE functionality.
  */
 &gpio_fwd {
 	nrf21540-gpio-if {
 	    gpios = <&gpio1 7 0>,
+		<&gpio1 6 0>,
```

Comment:
Is this the ANT_SEL pin? Was it missing in the old dts file?

Reply:
This is a mistake added here. This entry is not needed for radio_test.